src/admin-views/base_admin_panel.php

```diff
@@ -1,35 +1,39 @@
+<?php
+	$total_tickets = Tribe__Tickets__Tickets_Handler::instance()->get_total_event_capacity( $post_id );
+	$container_class = 'tribe_sectionheader ticket_list_container';
+	$container_class .= ( empty( $total_tickets ) ) ? ' _blank' : '' ;
```

Comment:
`empty(-1)` is boolean false. So it works as it sits but if it's clearer for others I can change it. :)